u4-databases/views/uni-iteration-details.hbs

```diff
@@ -0,0 +1,14 @@
+
+{{#>uni iteration }}
```

Comment:
คุณเข้าใจวิธีการทำงานนี้หรือไม่? มันใหม่สำหรับฉัน :)
Do you understand how this works? It was new to me :)